common/locales/en/quests.json

```diff
@@ -26,6 +26,8 @@
     "pending": "Pending",
     "questStart": "Once all members have either accepted or rejected, the quest begins. Only those that clicked \"accept\" will be able to participate in the quest and receive the drops. If members are pending too long (inactive?), the quest owner can start the quest without them by clicking \"Begin\". The quest owner can also cancel the quest and regain the quest scroll by clicking \"Cancel\".",
     "questStartBroken": "Once all members have either accepted or rejected, the quest begins... Only those that clicked \"accept\" will be able to participate in the quest and receive the drops... If members are pending too long (inactive?), the quest owner can start the quest without them by clicking \"Begin\"... The quest owner can also cancel the quest and regain the quest scroll by clicking \"Cancel\"...",
+    "questCollection": "+<%= val %> quest item found",
```

Comment:
Currently, yes, just one at a time. That might change in future but if it does, then we should introduce programmatic pluralisation. One of our users will be sure to remind us if we forget. :) "item" is good for now.

@khipkin It definitely seems wrong though at first glance, because of the variable. :)